### ISSUE TITLE
Add some "completed" state on playlist items

### ISSUE THREAD

**Issue by vpresecki** [scrimba/community#486]
It would be faster to navigate around courses if on playlist user could see what lectures he completed (checkbox, different color, label....). 
During the course when people want to go back on some lecture finding where they left can be quite hard and slow if you did not remember to check the name of the lecture before. It would be enough just to put some label/color anything eye-catchy so navigating through playlist could be a lot faster and would not slow down users.

**Comment by perborgen** [comment]
We have this. Doesn't it work on your end?

![Screenshot 2020-04-29 at 09.05.27.png](https://gitspeak.com/-/nUp4h5Lca16bb/Screenshot%202020-04-29%20at%2009.05.27.png)

**Comment by vpresecki** [comment]
Yes, that works but it's required to navigate back from the course or just have it in other tab/monitor and refresh i guess.
![playlistScrimba](https://user-images.githubusercontent.com/23663808/80569698-c26ce580-89f9-11ea-8128-c3707ba13fb3.png)

It would be much faster and user friendly to see where you left on the playlist in case you go back few lectures to remind yourself for something, it would also help not to break the rythm/focus of the student.

**Comment by perborgen** [comment]
Got it, we're definitely going to add progress indicators inside the casts as well.

**Comment by numanzahid** [comment]
@perborgen 
can we please have watch states on course box as well. 
something like this maybe.
![course-1](https://user-images.githubusercontent.com/6734385/80676866-1f879a80-8ad1-11ea-8ad7-549a28d04a4a.png)

**Comment by Ezra-Siton-UIX** [comment]
+1 for this idea (Very helpful)
